cmd/influxd/run/server_test.go

```diff
@@ -494,43 +495,10 @@ func TestServer_Write_LineProtocol_Integer(t *testing.T) {
 	}
 }
 
-// Ensure the server returns a partial write response when some points fail to parse. Also validate that
-// the successfully parsed points can be queried.
-func TestServer_Write_LineProtocol_Partial(t *testing.T) {
```

Comment:
This test shouldn't be removed.  It is testing that multiple points in a line protocol batch doesn't fail the whole batch if one fails to parse.